server/src/routes/token.ts

```diff
@@ -37,8 +37,11 @@ export function createTokenRoute(serverCredentials: ServerCredentials) {
      * Note: For internal use
      */
     if (
-      serverCredentials.ENABLE_ABOUT_PAGE === 'true' &&
-      !verifyTwilioEmail(userInfo.email)
+      typeof serverCredentials.EMAIL_VERIFICATION_REGEX !== 'undefined' &&
+      !verifyTwilioEmail(
+        userInfo.email,
+        new RegExp(serverCredentials.EMAIL_VERIFICATION_REGEX),
+      )
     ) {
       const msg = 'Must be a valid @twilio.com email';
```

Comment:
Let's reference the regex here, since the regex may not actually be `@twilio.com`.

Reply:
What should the message be then? 
`Does not match regex` ?

Reply:
Actually I made it dynamic instead, and then we can show this error message to the user for any email.